### PR TITLE
Open 14060-14079 on idr-omero-external

### PR DESCRIPTION
**Pull request by manics**
Opens port range 14060-14079 for future use with load-balancing

Tag: `1.2.0`

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,16 @@
     - 4063
     - 4064
 
+- name: OMERO external load balanced security group rules
+  os_security_group_rule:
+    direction: ingress
+    port_range_max: 14079
```
Comment (sbesson) [inline tasks/main.yml:26]: Would we want to make this range configurable?

Comment (manics) [reply tasks/main.yml:26]: In theory yes. In practice this role has a single use, and the rules are shared across all IDR deployments, so I think it's safest not to make this configurable.

**Comment by jburel** [comment]
Tested during docker and idr-py review
it works

**Comment by sbesson** [comment]
Thanks, merging and tagging as `1.2.0`.

**Pull request merged**